unlock-app/src/components/interface/checkout/alpha/Checkout/Returning.tsx

```diff
@@ -106,19 +106,19 @@ export function Returning({
               </Button>
             ) : (
               <div className="flex justify-between gap-4">
+                <Button className="w-full" onClick={() => onClose()}>
+                  Return
+                </Button>
                 {!lock?.isSoldOut && (
```

Comment:
What's interesting is that a sold out lock will still allow renewals so I think we can remove this